proxy/proxy.go

```diff
@@ -169,10 +169,11 @@ func InitPassiveHealthChecker(o map[string]string) (bool, *PassiveHealthCheck, e
 	}
 
 	result := &PassiveHealthCheck{}
-	keysInitialized := make(map[string]struct{})
+	requiredKeysInitialized := make(map[string]struct{})
```

Comment:
Maybe
```diff
diff --git a/proxy/proxy.go b/proxy/proxy.go
index 2de0161d..7c8b2e4d 100644
--- a/proxy/proxy.go
+++ b/proxy/proxy.go
@@ -16,6 +16,7 @@ import (
        "net/url"
        "os"
        "runtime"
+       "slices"
        "strconv"
        "strings"
        "time"
@@ -169,9 +170,10 @@ func InitPassiveHealthChecker(o map[string]string) (bool, *PassiveHealthCheck, e
        }
 
        result := &PassiveHealthCheck{}
-       keysInitialized := make(map[string]struct{})
+       requiredOptions := []string{"max-drop-probability"}
 
        for key, value := range o {
+               requiredOptions = slices.DeleteFunc(requiredOptions, func(s string) bool { return s == key })
                switch key {
                case "period":
                        period, err := time.ParseDuration(value)
@@ -212,13 +214,12 @@ func InitPassiveHealthChecker(o map[string]string) (bool, *PassiveHealthCheck, e
                default:
                        return false, nil, fmt.Errorf("passive health check: invalid parameter: key=%s,value=%s", key, value)
                }
-
-               keysInitialized[key] = struct{}{}
        }
 
-       if len(keysInitialized) != 4 {
-               return false, nil, fmt.Errorf("passive health check: missing required parameters")
+       if len(requiredOptions) != 0 {
+               return false, nil, fmt.Errorf("passive health check: missing required parameters %v", requiredOptions)
        }
+
        if result.MinDropProbability >= result.MaxDropProbability {
                return false, nil, fmt.Errorf("passive health check: minDropProbability should be less than maxDropProbability")
        }
```

Reply:
I like the code snippet. What about the exact name: I still would like some word after `required` to be specified, so I'll use `requiredParams`.